.github/ISSUE_TEMPLATE/docs_bug_report.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Documentation bug report
+about: Help us improve the documentation by keeping it typo free and up to date
```

Comment:
`typo-free`, perhaps, since we're using it as a compound adjective here? (Meta, I know)

Reply:
```suggestion
about: Help us improve the documentation by keeping it typo-free and up to date
```